### ISSUE TITLE
Showers and sinks are reversed

### ISSUE THREAD

**Issue by MrMelbert** [wall-nerds/wallening#248]
Showers and sinks use the wall mount directional helpers

https://github.com/wall-nerds/wallening/blob/2053fe63a88070896ac327707d97d3df982e53cd/code/game/objects/structures/shower.dm#L61

But in their actual sprite sheet, what you would consider `WEST` ("the shower is on the west wall") is actually `EAST` ("the direction the water comes out of the shower head flows east"). 
Same for sinks

![image](https://github.com/wall-nerds/wallening/assets/51863163/a63a4bbf-b687-419a-a50e-f1965cb1b806)

**Comment by MrMelbert** [comment]
LemonInTheDark you mark my issue as resolved yet I see this on live servers

![image](https://github.com/user-attachments/assets/a6146540-84c7-4c2e-a270-f0b571d9683d)

**Comment by LemonInTheDark** [comment]
GOD DAMN IT

**Comment by LemonInTheDark** [comment]
fixed question mark?

**Comment by unit0016** [comment]
Nope; still scuffed - sinks also appear to be backwards N/S; additionally
![image](https://github.com/user-attachments/assets/68aa69cd-63a4-4cf5-ab61-30e69fcd81c6)
![image](https://github.com/user-attachments/assets/d276649f-ecf9-4161-86b8-bb8dfa713b88)

**Comment by ArcaneMusic** [comment]
aaaaaaaa

Alright hold on
